README.md

```diff
@@ -1 +1,72 @@
 # babel-plugin-transform-react-pug
+
+The official means by which you can use pug in your react components, replaces the use of react-jade when moving to pug.
+
+This plugin transforms the pug inside of your react components.
+
+## Installation
+```
+npm install babel-plugin-transform-react-pug
+```
+## Usage
+.babelrc
+```js
+{
+  "presets": ["forbeslindesay"],
+  "plugins": [
+    "transform-react-pug",
+  ]
+}
+```
+## Examples
+
+### Example 1
+You can now create a react component with your pug inside it.
+```js
+import React from 'react';
+
+class MyComponent extends React.Component {
+
+  render() {
+    return pug`
+      div
+        h1 My Component
+        p This is my component using pug.
+    `;
+  }
+}
```

Comment:
You could do this as a "functional component":

function MyComponent() {
    return pug`
       div
         h1 My Component
         p This is my component using pug.
   `;
}
```
(you'll need to sort out my indentation).